client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.jsx

```diff
@@ -82,15 +77,15 @@ const PrePurchaseNotices = () => {
 
 		if ( ! planSlugInCart ) return null;
 
-		if ( isPlanIncludingSiteBackup( state, siteId, planSlugInCart ) ) {
+		if ( siteHasFeature( state, siteId, WPCOM_FEATURES_BACKUPS ) ) {
 			return currentSiteProducts.find( isJetpackBackup );
```

Comment:
Is this going to work? `siteHasFeature` is looking at the current site purchases, as is `currentSiteProducts`, ignoring the product that's in the cart. From what I can tell, all plans that pass the `planSlugInCart` test happen to include the three features that are being checked here which is why it's not noticeable. 

This should probably be a `planHasFeature` check instead?

Reply:
I agree! I won't work. I used `siteHasFeature` because the code returns true only when the feature is already available on the site.